src/main/kotlin/org/rust/ide/formatter/processors/RsStatementSemicolonFormatProcessor.kt

```diff
@@ -10,37 +10,44 @@ import com.intellij.openapi.util.TextRange
 import com.intellij.psi.PsiElement
 import com.intellij.psi.PsiRecursiveElementVisitor
 import com.intellij.psi.impl.source.codeStyle.PreFormatProcessor
+import org.rust.lang.core.psi.*
 import org.rust.lang.core.psi.RsElementTypes.SEMICOLON
-import org.rust.lang.core.psi.RsMatchArm
-import org.rust.lang.core.psi.RsPsiFactory
-import org.rust.lang.core.psi.RsRetExpr
 import org.rust.lang.core.psi.ext.elementType
 import org.rust.lang.core.psi.ext.getNextNonCommentSibling
-import java.util.*
 
-class RsReturnStatementFormatProcessor : PreFormatProcessor {
+class RsStatementSemicolonFormatProcessor : PreFormatProcessor {
     override fun process(element: ASTNode, range: TextRange): TextRange {
         if (!shouldRunPunctuationProcessor(element)) return range
 
-        val returnElements = ArrayList<RsRetExpr>()
+        val returns = arrayListOf<RsRetExpr>()
```

Comment:
Hmm, I think we don't need 3 different lists here, we can just leave a single list of `RsExpr` type